test/integration/controllers/lesson.spec.js

```diff
@@ -0,0 +1,65 @@
+const { serverCleanup, serverInit, stopServer } = require('~/test/setup')
+const { expectError } = require('~/test/helpers')
+const { UNAUTHORIZED } = require('~/consts/errors')
+const testUserAuthentication = require('~/utils/testUserAuth')
+
+const endpointUrl = '/lessons'
+
+let mockUploadFile = jest.fn().mockResolvedValue('mocked-file-url')
+
+const testLesson = {
+  title: 'title',
+  description: 'description',
+  attachments: [
+    {
+      src: 'data:image/jpeg;base64,/9j/4AAQSkZJRgABAgAAAQABAAD...',
+      name: 'example1.jpg'
+    },
+    {
+      src: 'data:image/jpeg;base64,/9j/4AAQSkZJRgABAgAAAQABAAD...',
+      name: 'example2.jpg'
+    }
+  ]
+}
+
+describe('Lesson controller', () => {
+  let app, server, accessToken, testLessonResponse
+
+  beforeAll(async () => {
+    ;({ app, server } = await serverInit())
+  })
+
+  beforeEach(async () => {
+    accessToken = await testUserAuthentication(app, { role: 'tutor' })
+
+    const uploadService = require('~/services/upload')
```

Comment:
shouldn't it be above with other imports?

Reply:
thanks for catching that. Fixed